beekeeper/src/main/java/io/servicecomb/company/beekeeper/BeekeeperApplication.java

```diff
@@ -25,7 +26,8 @@
 @SpringBootApplication
 public class BeekeeperApplication {
 
-  public static void main(String[] args) {
+  public static void main(String[] args) throws Exception {
+    Log4jUtils.init();
```

Comment:
this will cause error in log, since spring boot repackaging will generate a fat jar. it's not possible to merge log files in fat jars. the design of this Log4jUtils is flawed and not recommended to use here